ci/jenkins/jobs/projects-lab.yaml

```diff
@@ -713,6 +986,55 @@
                       default-excludes: true
                       fingerprint: false
                       only-if-success: false
+      - '{name}-{test_name}-no-scm':
+            test_name: windows-containerd-e2e-pending-label
```

Comment:
Please remove 'containerd' in windows job name because we have removed docker testbed.